cmd/fleetctl/preview.go

```diff
@@ -42,20 +45,19 @@ This command will create a directory fleet-preview in the current working direct
 				return err
 			}
 
-			// Download files if necessary
+			// Download files every time to ensure the user gets the most up to date versions
 			previewDir := previewDirectory()
-			if _, err := os.Stat(
-				filepath.Join(previewDir, "docker-compose.yml"),
-			); err != nil {
-				fmt.Printf("Downloading dependencies into %s...\n", previewDir)
-				if err := downloadFiles(); err != nil {
-					return errors.Wrap(err, "Error downloading dependencies")
-				}
+			fmt.Printf("Downloading dependencies into %s...\n", previewDir)
+			if err := downloadFiles(); err != nil {
+				return errors.Wrap(err, "Error downloading dependencies")
 			}
 
 			if err := os.Chdir(previewDir); err != nil {
 				return err
 			}
+			if _, err := os.Stat("docker-compose.yml"); err != nil {
+				return errors.Wrap(err, "docker-compose file not found in preview directory")
```

Comment:
cool to see how golang does this.  Used to taking errors to ["flavortown"](https://www.npmjs.com/package/flaverr) in js